docs/internals/contributing/new-contributors.txt

```diff
@@ -96,40 +101,45 @@ some advice to make your work on Django more useful and rewarding.
   recently involved in a discussion, then a ticket may not have the support
   required to get into Django.
 
-* **Start small**
+Start small
+-----------
 
   It's easier to get feedback on a little issue than on a big one. See the
   `easy pickings`_.
 
-* **If you're going to engage in a big task, make sure that your idea has
-  support first**
+If you're going to engage in a big task, make sure that your idea has support first
+-----------------------------------------------------------------------------------
 
   This means getting someone else to confirm that a bug is real before you fix
   the issue, and ensuring that there's consensus on a proposed feature before
   you go implementing it.
 
-* **Be bold! Leave feedback!**
+Be bold! Leave feedback!
+------------------------
 
   Sometimes it can be scary to put your opinion out to the world and say "this
   ticket is correct" or "this patch needs work", but it's the only way the
   project moves forward. The contributions of the broad Django community
   ultimately have a much greater impact than that of any one person. We can't
   do it without **you**!
 
-* **Err on the side of caution when marking things Ready For Check-in**
+Err on the side of caution when marking things Ready For Check-in
```

Comment:
@sarahboyce 
This doesn't look long in code but on the web page it is on two lines. Should we also shorten this heading as well?

![heading2lines](https://github.com/user-attachments/assets/886a10c2-8aff-4baf-aaff-c1da93989ae2)

Reply:
I agree, maybe something like `Be careful setting tickets as "Ready For Check-in"`?

Also, I know you pushed up changes but they look to be lost, can you push up again? :pray: